packages/swapper/src/swappers/zrx/ZrxSwapper.ts

```diff
@@ -84,10 +84,12 @@ export class ZrxSwapper implements Swapper {
 
   filterBuyAssetsBySellAssetId(args: BuyAssetBySellIdInput): AssetId[] {
     const { assetIds, sellAssetId } = args
-    return assetIds.filter((id) => id.startsWith('eip155:1') && sellAssetId.startsWith('eip155:1'))
+    return assetIds?.filter(
```

Comment:
this will return undefined whereas the function sig implies we would always like to return an array.

some other options:

default value - `const { assetIds = [], sellAssetId } = args`
nullish operator - `return (assetIds ?? []).filter` or `return assetIds?.filter() ?? []`

Reply:
good feedback, thanks